src/ralph/trade_marks/admin.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.admin import RalphAdmin, RalphTabularInline, register
+from ralph.admin.filters import DateListFilter
+from ralph.admin.views.extra import RalphDetailViewAdmin
+from ralph.attachments.admin import AttachmentsMixin
+from ralph.trade_marks.forms import IntellectualPropertyForm
+from ralph.trade_marks.models import (
+    ProviderAdditionalMarking,
+    TradeMarks,
+    TradeMarksLinkedDomains
+)
+
+
+class TradeMarksLinkedView(RalphDetailViewAdmin):
+    icon = 'table'
+    name = 'table'
+    label = _('Assigned to domain')
+    url_name = 'assigned-to-domain'
+
+    class TradeMarksLinkedInline(RalphTabularInline):
+        model = TradeMarksLinkedDomains
+        raw_id_fields = ('domain', admin.RelatedFieldListFilter)
+        extra = 1
+
+    inlines = [TradeMarksLinkedInline]
+
+
+@register(TradeMarks)
+class IntellectualPropertyAdmin(AttachmentsMixin, RalphAdmin):
```

Comment:
so is it IntelectualProperty admin or TradeMark admin?

Reply:
Changed into TradeMarkAdmin